index.js

```diff
@@ -119,7 +120,13 @@ module.exports = {
       }]
     });
 
-    const trees = [contentTree, pageTree, authorTree];
+    const includedTree = new AddIncludedJson(MergeTrees([contentTree, pageTree, authorTree]), {
```

Comment:
I think we might be able to simplify this a little bit if we define the API 🤔 

I think we could have it **replace** the content tree, something like this: 

```
let contentTree = doNormalStuff();

contentTree = new AddIncludedJson(contentTree, {
  sourceTrees: [authorTree],
  include: {
    content: ['author'],
  }
})
```

this essentially makes sense because we want to go through every entry in the content tree and add in thing defined in the include and we can look in the sourceTrees for those objects. This should prevent your need for the override on line 148

Reply:
I like this! Your initial description gave me the impression you wanted to pass in _all_ the trees at the same time, but I indeed also think this is better.

Reply:
I think it's ok for us to pass in the trees that we know are relevant to the includes 🤔 if someone did need everything then they could include everything 😂